src/structures/RichEmbed.js

```diff
@@ -284,6 +309,14 @@ class RichEmbed {
       } : null,
     };
   }
+
+  static checkField(name, value, inline = false) {
+    name = util.resolveString(name);
```

Comment:
there is a `resolveString` function declared at the bottom of the file, perhaps use that or switch the other calls to use util.resolveString and remove the function declared at the bottom